cmd/options.go

```diff
@@ -128,7 +128,7 @@ func getOptions(flags *pflag.FlagSet) (lib.Options, error) {
 		if err != nil {
 			return opts, err
 		}
-		opts.SystemTags = lib.GetTagSet(systemTagList...)
+		opts.SystemTags = stats.ToSystemTagSet(systemTagList)
```

Comment:
I think there should be some error checks here - if I execute `k6 run --system-tags=invalid,url`, k6 would currently ignore the `invalid` value

Reply:
I think it's the same behavior as before?

Reply:
Hmm that's roughly true - there's a difference, but it's internal, the users won't be able to notice it. Before this PR, with the `string`-y system tags, we didn't actually have a list of all possible system tag values to validate against, and now we do. So previously whatever tags users used would just get saved in the `systemTags` map, but any non-system ones were never checked, so they didn't influence the k6 behavior in any way. Now, we're silently discarding any invalid values - same user-facing behavior, different implementation details.

Even so, I'd argue that when users specify invalid configuration, aborting with an error or showing a warning at the very least is better UX. Imagine that users have a typo in the config, for example `k6 run --system-tags=url,pfoto,status script.js`. When the test runs without a hitch, it may take them quite a while to realize why they're not seeing the `proto` metric tags... :smile: 

That said, I'm fine with leaving this improvement for a separate PR. I'll add a note about this, and about the necessity for using pointers in order to know if users specified the value, in https://github.com/loadimpact/k6/issues/883. These things would have to be fixed when we improve the configuration... For now, can you just add something like a `//TODO: emit error instead of discarding invalid values` in `stats.ToSystemTagSet()` where we discard the invalid tag values?

Reply:
https://github.com/loadimpact/k6/issues/883#issuecomment-542151237